rllib/policy/tests/test_sample_batch.py

```diff
@@ -372,14 +372,15 @@ def test_to_device(self):
         s = SampleBatch(
             {
                 "a": np.array([1, 2]),
-                "b": {"c": (np.array([4, 5]), np.array([5, 6])), "e": [{"f": None}]},
```

Comment:
@sven1977 I could not think of a use case for a None here and neither could Kourosh. Please ack.

Reply:
Allowing Nones makes counting logic more convoluted and I believe it should simply not be supported.
We want to enforce tensors here in the long run. Wonky stuff can go into INFOS.

Reply:
(Not allowing Nones here does not break our tests)